src/main/kotlin/no/nav/syfo/api/endpoints/VedtakEndpoints.kt

```diff
@@ -37,7 +40,17 @@ fun Route.registerVedtakEndpoints(
             val navIdent = call.getNAVIdent()
             val callId = call.getCallId()
 
-            call.respond(HttpStatusCode.Created)
+            val newVedtak = vedtakService.createVedtak(
+                personident = personident,
+                veilederident = navIdent,
+                begrunnelse = requestDTO.begrunnelse,
```

Comment:
Blir etter hvert også `behandlerRef` og `behandlerDocument` inn her så mulig det er bedre å sende hele dtoen inn eller lage domeneobjekter her ute - hva tenker dere?

Reply:
Ser fint ut nå, så kan vurdere det når det dukker behov for flere parametere her.